src/loaders/collection-loader.coffee

```diff
@@ -4,6 +4,7 @@ _ = require 'underscore'
 Collection = require '../collection'
 AbstractLoader = require './abstract-loader'
 
+knownResponseKeys = ['count', 'page_count', 'page_number', 'results']
```

Comment:
`page_size`?

Reply:
I actually left that out of the brainstem PR, even though it's in the RFC. We don't actually need the page size for any calculations right now, I think.

Reply:
Food for thought: we should consider nesting these in `meta` for easier future refactorings. Though that is definitely a breaking API change.